src/platform/i386/timer.c

```diff
@@ -54,7 +54,7 @@
 #define TN_FSB_EN_CNF	(1ll << 14)	/* 1 = deliver interrupts via FSB instead of APIC */
 #define TN_FSB_INT_DEL_CAP	(1ll << 15)	/* read only, 1 = FSB delivery available */
 
-#define HPET_INT_ENABLE 0x3	/* Sets ENABLE_CNF and LEG_RT_CNF */
+#define HPET_INT_ENABLE(n) (0x1 << n)	/* Clears the INT n for level-triggered mode. */
```

Comment:
No, I didn't speak to him. I went through the HPET Specification document and made the changes. His comments don't seem right, though it is doing what it had to. As of now, we use Periodic timer in timer[0], so it sets that bit 0 for clearing INT 0. It also sets Bit 1 but that doesn't matter as we don't use it. So it works. I modified it to set the bit that corresponds to the timer that we use.